client/Components/Map.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { GoogleMap, LoadScript, Marker } from '@react-google-maps/api';
+import {TICKETMASTERAPIKEY, REACT_APP_GOOGLEAPIKEY} from '../secret'
+import axios from 'axios'
+class Map extends React.Component{
+  constructor(){
```

Comment:
prof never taught you this in lecture videos so i understand but always pass in `props` to `super`
```js
constructor(props) {
   super(props);
}
```